source/opt/inline_pass.cpp

```diff
@@ -438,7 +436,7 @@ void InlinePass::GenInlineCode(
         // Copy remaining instructions from caller block.
         auto cii = call_inst_itr;
         for (++cii; cii != call_block_itr->end(); ++cii) {
-          std::unique_ptr<ir::Instruction> cp_inst(new ir::Instruction(*cii));
+          std::unique_ptr<ir::Instruction> cp_inst(cii->Clone());
```

Comment:
Marking the Instruction copy constructor with =delete() wreaks havoc atm, right?

Reply:
Yes, it does.  I will eventually do that.  I think I need to change the container for the debug instruction inside the Instruction class, and we should be good.